libchannelserver/ChannelSession.h

```diff
@@ -132,6 +132,14 @@ class ChannelSession : public std::enable_shared_from_this<ChannelSession>
         m_networkStat = _handler;
     }
 
+    void setRemotePublicKey(dev::h512 const& _remotePublicKey)
+    {
+        CHANNEL_SESSION_LOG(INFO) << LOG_DESC("setRemotePublicKey: set sdk public key")
+                                  << LOG_KV("sdkPublicKey", _remotePublicKey.abridged());
+        WriteGuard l(x_remotePublicKey);
```

Comment:
why need a mutex?

Reply:
You are right, the mutex is not required. I have already deleted it.